molgenis-app-vibe/pom.xml

```diff
@@ -135,7 +135,7 @@
   <dependencies>
     <dependency>
       <groupId>org.molgenis</groupId>
-      <artifactId>vibe</artifactId>
+      <artifactId>vibe-core</artifactId>
```

Comment:
Are the exclusions still required?

Reply:
will be resolved in new vibe-core version. @svandenhoek can you create an issue in vibe repo?

Reply:
Done: https://github.com/molgenis/vibe/issues/57